src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/VilkårperiodeGrunnlagService.kt

```diff
@@ -110,6 +110,8 @@ class VilkårperiodeGrunnlagService(
     private fun antallMånederBakITiden(behandling: Saksbehandling): LocalDate {
```

Comment:
Javadocen til denne funksjonen er 
```
    /**
     * Vid en førstegangsbehandling skal man bruke mottatt tidspunkt minus antall måneder for gitt stønadstype
     * Når man revurderer skal man hente grunnlag fra og med datoet man revurderer fra, uavhengig når man søker fra
     */
```
Siste setning må endres dersom vi går for denne løsningen :)